ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/validation/RequestValidator.java

```diff
@@ -113,6 +113,7 @@ public int validate(
 
     final KsqlExecutionContext ctx = requireSandbox(snapshotSupplier.apply(serviceContext));
     final Injector injector = injectorFactory.apply(ctx, serviceContext);
+    final KsqlConfig ksqlConfig = ksqlEngine.getKsqlConfig();
```

Comment:
don't use a passed in engine here. instead, get the config out of the sandbox (`ctx`). You'll have to add an interface for getting the config (see my other comment).